src/modules/auth/auth.controller.ts

```diff
@@ -36,4 +37,18 @@ export class AuthController {
   async signUp(@Body() authSignUpDto: AuthSignUpDto): Promise<AuthResponse> {
     return this.authService.signUp(authSignUpDto);
   }
+
+  @PublicRoute()
+  @Post('generate-access')
+  async generateAccess(
+    @Body() authGenerateAccessDto: AuthGenerateAccess,
+  ): Promise<AuthTokens> {
+    return this.authService.generateAccess(authGenerateAccessDto);
+  }
+
+  @Get('current')
+  @HttpCode(200)
+  getCurrent(@GetUser() user: User): User {
```

Comment:
it should be in user module and this data should be protected

Reply:
![image](https://github.com/ZenBit-Tech/sunny-bunnies_be/assets/87016333/5406eb79-60e1-4943-bae1-f547235d4531)
route is protected